app/src/main/java/ch/epfl/sweng/swenggolf/Leaderboard.java

```diff
@@ -0,0 +1,101 @@
+package ch.epfl.sweng.swenggolf;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ch.epfl.sweng.swenggolf.database.AttributeOrdering;
+import ch.epfl.sweng.swenggolf.database.Database;
+import ch.epfl.sweng.swenggolf.database.DbError;
+import ch.epfl.sweng.swenggolf.database.ValueListener;
+import ch.epfl.sweng.swenggolf.profile.User;
+import ch.epfl.sweng.swenggolf.tools.FragmentConverter;
+
+public class Leaderboard extends FragmentConverter {
+
+    private LeaderboardAdapter mAdapter;
+    protected RecyclerView.LayoutManager mLayoutManager;
+    private TextView errorMessage;
+    private TextView noUser;
+    public static final List<User> userList = new ArrayList<>();
+    private static final String ATTRIBUTE = "points";
```

Comment:
I define such a constant in UserDatabase, maybe you might use the one I created ?

Reply:
Yes thank you, helpful!